conans/model/build_info.py

```diff
@@ -12,6 +13,50 @@
 DEFAULT_FRAMEWORK = "Frameworks"
 
 
+class DefaultOrderedDict(OrderedDict):
+    # Source: http://stackoverflow.com/a/6190500/562769
```

Comment:
This is probably conditioned to the original PSF license of http://code.activestate.com/recipes/523034-emulate-collectionsdefaultdict/.

I am a bit reluctant to use a fully fledged DefaultOrderedDict for this, as it might be abused, we are not sure what methods are actually in use and what not. I would prefer a class, no inheritance, with just a ``__getitem__`` attribute, and an OrderedDict inside as implementation detail.

Reply:
Yes, this was temporarily done to get rid of the issue without wasting time in the implementation. I will follow your approach. Thanks